apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -0,0 +1,105 @@
+/** @file Settings menu cog icon */
+import React, {Component} from 'react';
+import Radium from 'radium';
+import Portal from 'react-portal';
+import msg from '@cdo/locale';
+import FontAwesome from '../../templates/FontAwesome';
+import color from '../../util/color';
+import SettingsMenu from './SettingsMenu';
+
+const style = {
+  iconContainer: {
+    float: 'right',
+    marginRight: 10,
+    marginLeft: 10,
+    height: '100%',
+    cursor: 'pointer',
+    color: color.lighter_purple,
+    ':hover': {
+      color: color.white,
+    }
+  },
+  assetsIcon: {
+    fontSize: 18,
+    verticalAlign: 'middle',
+  },
+};
+
+class SettingsCog extends Component {
+  constructor(props) {
+    super(props);
+
+    // Autobind
```

Comment:
I'm not sure.  [The _auto-_ root](http://www.dictionary.com/browse/auto-) meaning "self" doesn't always imply "automatic" - I assumed it might be equally valid to refer to a class explicitly binding its methods to itself as "autobinding."  Do you think the comment makes this more or less confusing?